packages/amplify-category-predictions/src/provider-utils/awscloudformation/prediction-category-walkthroughs/identify-walkthrough.js

```diff
@@ -137,29 +134,43 @@ async function configure(context, predictionsResourceObj, configMode /*add/updat
 
   // only ask this for add
   if (!parameters.resourceName) {
-    const identityAssetTypeInput = identifyAssets.setup.type();
-    answers[identityAssetTypeInput.name] = await prompter.pick(identityAssetTypeInput.message, identityAssetTypeInput.choices);
+    answers['identifyType'] = await prompter.pick(
```

Comment:
nit: `answers.identifyType` (also applies to other places where a string literal is used as a dynamic key)